### ISSUE TITLE
Accepting an assignment creates multiple repositories (e.g., username, username-1, username-2)

### ISSUE THREAD

**Issue by botterweck** [education/classroom#1206]
When accepting assignments, students often end up with multiple repositories with URLs ending in username, username-1, username-2 and so on.

For instance, to test I just tried to accept an assignment (which I created myself). Initially, I got the following message, notice the incomplete URL after "created here".

![image](https://user-images.githubusercontent.com/1396137/32026606-00ed9f6a-b9dd-11e7-98b6-002bc0b6f0e4.png)

Upon clicking on the hashtag, I then got the similar page but now with a complete link (I did not screenshot that), which pointed me to 
https://github.com/ul-csis/cs4028-assignment-02-botterweck-2
and not the expected
https://github.com/ul-csis/cs4028-assignment-02-botterweck

On checking the organization page (https://github.com/ul-csis/) I saw that we now have
https://github.com/ul-csis/cs4028-assignment-02-botterweck
https://github.com/ul-csis/cs4028-assignment-02-botterweck-1
https://github.com/ul-csis/cs4028-assignment-02-botterweck-2

On the assignment page (https://classroom.github.com/classrooms/32250248-cs4028-e-business-architectures/assignments/cs4028-assignment-02)
the repo with the highest number is linked for the student.

However, in the past that was not reliable. We had cases where repositories were created (could be found on the organization's page) but where not linked from the assignment page.

**Comment by athityakumar** [comment]
This feature of multiple forks for the same pair of repository and user account (with suffixes like -1, -2, etc.) lies within GitHub's core. As GitHub Education is inheriting this from the core, I'm not sure whether it can be fixed within this project.

I'm not a maintainer, I'm just an open-source contributor who wants to provide a quick and rough idea of what's going on here. Of course, one of the maintainers should be able to respond to your query better. 😄

**Comment by botterweck** [comment]
I understand that it should **be possible** that the same user creates multiple independent forks of the same repository.

However, it should not be the default behavior that accepting an assignment leads to unpredictable results. 

I could understand that multiple repositories are created if the student (for whatever reason) activates the invitation link multiple times.

However, 

1. It seems that the acceptance procedure is not reliable (I got the screenshot that I reported above on my first click on the invitation link).
2. It seems that the acceptance is completed, but the student gets an repository (with a number after it) whereas that last repository is not listed in the assignment page of the class instructor.
